.env

```diff
@@ -0,0 +1 @@
+CLIENT_ID=2324024081-2nk8c6amog25npupplbmaneaj35ot6c7.apps.googleusercontent.com
```

Comment:
It's a mistake to store .env file in public repository. Convert it to .env.example file. It should be better to give us a token by Skype chat.